### PR TITLE
Alter sms campaign slack notification

### PR DESCRIPTION
**Pull request by oakinh**

### CODE REVIEW

File: api/controllers/voterData/voterFile/schedule.js [hunk 1]
```diff
@@ -42,21 +54,66 @@ module.exports = {
...
     try {
-      const { budget, audience, script, date, message, voicemail } = inputs;
+      const { budget, audience, script, date, message, voicemail, voterFileUrl, type } = inputs;
       const { user } = this.req;
-      const { firstName, lastName, email } = user;
+      const { firstName, lastName, email, phone } = user;
+      const campaign = await sails.helpers.campaign.byUser(user.id);
+      const crmCompany = await sails.helpers.crm.getCompany(campaign);
+      const assignedPa = await getCrmCompanyOwnerName(crmCompany);
+      const aiGeneratedScript = sanitizeHtml(campaign.aiContent[script].content, {
```
Comment (RavenHursT) [inline api/controllers/voterData/voterFile/schedule.js:63]: Prolly good to put some existential checks here..

File: api/controllers/voterData/voterFile/schedule.js [hunk 2]
```diff
@@ -42,21 +54,66 @@ module.exports = {
...
+        allowedTags: [],
+        allowedAttributes: {},
+      });
+
+      if (VoterFileUrl && !isUrl(VoterFileUrl)) { // This WILL filter out localhost URLs, comment out if localhost testing
```
Comment (RavenHursT) [inline api/controllers/voterData/voterFile/schedule.js:68]: 😬😬😬

There has got to be a better way to do this.. how about adding `&& !VoterFileUrl.startsWith('http:localhost')` to this condition?

Comment (RavenHursT) [reply api/controllers/voterData/voterFile/schedule.js:68]: Also, are you sure this worked? 

You have `VoterFileUrl` here, but on line 57 above the variable is `voterFileUrl`? 🤔

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "@mapbox/mapbox-sdk": "^0.15.5",
     "@pinecone-database/pinecone": "^1.1.2",
     "@sailshq/lodash": "^3.10.3",
+    "@slack/web-api": "^7.5.0",
```
Comment (RavenHursT) [inline package.json:20]: Did you end up using this? I'm not seeing a `require` for it anywhere? 🤔

**Review by RavenHursT — CHANGES_REQUESTED** [review]

**Review by RavenHursT — APPROVED** [review]

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**